akka-docs/rst/java/code/docs/pattern/BackoffSupervisorDocTest.java

```diff
@@ -8,6 +8,7 @@
 import akka.pattern.BackoffSupervisor;
 import akka.testkit.TestActors.EchoActor;
 //#backoff-imports
+import docs.AbstractJavaTest;

```

Comment:
And not at all here actually because, even though the class name is suffixed with `Test` it isn't at all a test...